src/drive/web/modules/filelist/HammerComponent.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react'
 import Hammer from '@egjs/hammerjs'
 import propagating from 'propagating-hammerjs'
+
```

Comment:
can be removed, not needed?

Reply:
It's to separate the import and the component definition